tests/jwt.test.js

```diff
@@ -0,0 +1,82 @@
+import { login, loginVerify } from '../src/auth'
+import jwt from 'jsonwebtoken'
+import { jest } from '@jest/globals';
+
+const originalJwtSign = jwt.sign
+
+describe('login', () => {
+    let mockRequest
+    let mockResponse
+
+    beforeEach(() => {
+        mockRequest = {
+            body: {username: 'testUser'}
+        }
+        mockResponse = {
+            cookie: jest.fn(),
+            redirect: jest.fn()
+        }
+
+        jwt.sign = jest.fn().mockReturnValue('testToken')
+    })
+
+    afterEach(() => {
+        jwt.sign = originalJwtSign
```

Comment:
There are utilities in `jest` for creating mocks and resetting them, so that you don't have to implement these things yourself. But what you've done here is still neat.